css/desktop-layout.css

```diff
@@ -0,0 +1,433 @@
+@media only screen and (min-width: 768px) {
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: inter, Arial, Verdana, Tahoma, sans-serif;
+    list-style: none;
+  }
```

Comment:
- [ ] Nice job using more media queries to manage your layout, But As I can see your design break's for screen size between `601px` - `767px`. Please make sure you have responsive design in different screen sizes which includes these screens. 
![Untitled](https://user-images.githubusercontent.com/69077061/213260386-1ece78e2-692b-4645-bfdd-8fc49f7ab566.png)